en/sim_sih/index.md

```diff
@@ -164,6 +164,45 @@ export PX4_HOME_ALT=28.5
 make px4_sitl sihsim_quadx
 ```
 
+
+### Adding new airframes
+
+[Adding a new airframe](../dev_airframes/adding_a_new_frame.md) for use in SIH simulation is much the same as for other use cases.
+You still need to configure your vehicle type and [geometry](../config/actuators.md) (`CA_` parameters) and start any other defaults for that specific vehicle.
+
+The specific differences for SIH simulation airframes are listed in the sections below.
+
+For all variants of SIH:
+
+  - Set all the [Simulation In Hardware](../advanced_config/parameter_reference.md#simulation-in-hardware) parameters (prefixed with `SIH_`) in order to configure the physical model of the vehicle.
+  - `param set-default SYS_HITL 2` to enable SIH on the next boot.
+  - `param set-default CBRK_SUPPLY_CHK 894281` to disable power valid check.
+  - `param set-default CBRK_IO_SAFETY 22027` to disable IO safety check.
+  - `param set-default EKF2_GPS_DELAY 0` to improve state estimator performance (the assumption of instant GPS measurements would normally be unrealistic, but is accurate for SIH).
+
+For SIH on FC: 
+
+  - Airframe file goes in `ROMFS/px4fmu_common/init.d/airframes` and follows the naming template `${ID}_${model_name}.hil`, where `ID` is the `SYS_AUTOSTART_ID` used to select the airframe, and `model_name` is the airframe model name.
+  - Add the model name in `ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt` to generate a corresponding make target. 
+  - Actuators are configured with `HIL_ACT_FUNC*`, rather than `PWM_MAIN_FUNC*` as usual.
+    This is to avoid using the actual actuators in SIH.
+    Similarly, the bitfield for inverting individual actuator output ranges is `HIL_ACT_REV`, rather than `PWM_MAIN_REV`.
+  - `param set CBRK_USB_CHK 894281` to disable USB link check. 
+
+For SIH as SITL (no FC):
+
+ - Airframe file goes in `ROMFS/px4fmu_common/init.d-posix/airframes` and follows the naming template `${ID}_sihsim_${model_name}`, where `ID` is the `SYS_AUTOSTART_ID` used to select the airframe, and `model_name` is the airframe model name.
+ - Add the model name in `src/modules/simulation/simulator_sih/CMakeLists.txt` to generate a corresponding make target.
+ - Actuators are configured as usual with `PWM_MAIN_FUNC*` and `PWM_MAIN_REV`.
+ - Additionally, set:
+   - `PX4_SIMULATOR=${PX4_SIMULATOR:=sihsim}`
+   - `PX4_SIM_MODEL=${PX4_SIM_MODEL:=svtol}` (replacing `svtol` with the airframe model name)
+   - `param set-default SENS_EN_GPSSIM 1`
+   - `param set-default SENS_EN_BAROSIM 1`
+   - `param set-default SENS_EN_MAGSIM 1`
```

Comment:
```suggestion
```

Reply:
This comes back to https://github.com/PX4/PX4-Autopilot/commit/5676cc32bc3651ee1f65a918d9d681232c67e995
So all these parameters turn on virtual sensor drivers that take the ground truth data from the simulator (SIH) and publish noisy simulated sensor data to the topics normally used by the PX4 pipeline. As I understand if the parameters are not set to 1 then you don't get any simulated sensor data fed to PX4.